runtime/src/accounts_db.rs

```diff
@@ -7175,6 +7175,8 @@ impl AccountsDb {
             timings.report();
         }
 
+        self.accounts_index.log_secondary_indexes();
```

Comment:
Maybe it should be optional? Bit worried it will be huge on a mainnet snapshot. Or maybe just print the top 20?

Reply:
Limited to top 20. it logs nothing if there are no secondary indexes.